packages/unity-bootstrap-theme/stories/atoms/video/video.js

```diff
@@ -1,17 +1,26 @@
 import { googleAnalytics } from "../../../src/js/googleAnalytics";
 
 export const initVideo = () => {
+  // constants
+  const DOM_ELEMENT_VIDEO = "video";
+  const DOM_ELEMENT_VIDEO_BTN = ".uds-video-btn-play";
+  const DOM_ELEMENT_VIDEO_OVERLAY = ".uds-video-overlay";
+  const EVENT_CLICK = "click";
+  const EVENT_ENDED = "ended";
+  const STYLE_DISPLAY_FLEX = "flex";
+  const STYLE_DISPLAY_NONE = "none";
+
   // get elements to interact with
-  const videoPlayer = document.querySelector("video");
-  const videoOverlay = document.querySelector(".uds-video-overlay");
-  const videoBtnPlay = document.querySelector(".uds-video-btn-play");
+  const $videoPlayer = document.querySelector(DOM_ELEMENT_VIDEO);
```

Comment:
Is there a possibility of more than one video element in the document?

Reply:
I don't know about that possibility